legend-engine-pure-code-compiled-core/src/main/resources/core/pure/mutation/mutation.pure

```diff
@@ -0,0 +1,16 @@
+import meta::pure::mapping::*;
+import meta::pure::runtime::*;
+import meta::pure::graphFetch::*;
+
+function meta::pure::mutation::execution::save<T>(type: Class<T>[1], rootGraphFetchTree: RootGraphFetchTree<T>[1], values: T[*], mapping: Mapping[1], runtime: Runtime[1]): T[*]

```

Comment:
We should add `<<functionType.NotImplementedFunction>>` to this function.

Reply:
Thanks. Done